spec/controllers/sessions_controller_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
+
+describe SessionsController do
+
+  describe "GET new" do
+    it "renders new template for unauthenticated users" do
```

Comment:
You are testing rails here, in `new` action you are testing whether it renders `:new` template.